hasher-matcher-actioner/CONTRIBUTING.md

```diff
@@ -209,3 +209,25 @@ $ hmacli run-api
 ```
 
 This will run the API on port 8080 on localhost.
+
+## Troubleshooting
+
+### Default devcontainer.json
+- Context:
+    - You clone the `ThreatExchange` repo and open the `hasher-matcher-actioner/` directory in a VSCode remote container.
+    - You select VSCode to use a preexisting `Dockerfile`
+    - The VSCode console logs show
+        ```bash
+        [2022-01-24T14:14:49.974Z] /bin/sh: line 26: tar: command not found
+        [2022-01-24T14:14:49.974Z] Exit code 127
+        ```
+- Cause: You did not create a `devcontainer.json` before opening the directory in the remote container. This caused VSCode to create a default `devcontainer.json` which points to the wrong `Dockerfile`.
```

Comment:
doesn't your change above prevent this now?

Reply:
Yes, the change to rename `devcontainer.json` will prevent this particular problem. I left the documentation for this problem partly to explain why the renaming was done, and partly because if future features make `devcontainer.json` not work out of the box, we could be back to using `devcontainer.json.example` which will reopen the possibility of this problem.

Reply:
ahh, I feel like adding a reference to `devcontainer.json.example` while also removing it is a tad questionable. 

We could bring it back but that is when I would add this line. Could we either keep around a copy of `devcontainer.json.example` or instead point them to the new `devcontainer.json`

Reply:
That makes sense. I've removed the reference to `devcontainer.json.example` since the file has been deleted.

Reply:
Awesome, I'll go ahead and merge. Thanks again for adding this